spring-kafka/src/main/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainer.java

```diff
@@ -156,15 +169,16 @@ protected void doStart() {
 				this.concurrency = this.partitions.length;
 			}
 			setRunning(true);
+			logger.info("Topics from ConcurrentMessageListenerContainer-->:" + (this.topics));
```

Comment:
Thank you, @kvmkreddy, for contribution!

Everything looks good and really is ready for merge.
Although I have several simple concerns. 
So, let me express them with the hope that you'll share your mind.

What is the point of this `logger.info()` for `this.topics`?
I think we still can live without it. 
The `this.topics` may be `null` and even it is able to be changed at runtime, it does not make much sense to log that anyway. Just because this value isn't changed internally without an external end-user involvement.

More over we don't use `-->` formatting in the Framework logs, plus it looks like you should have a whitespace after colon :smile: